tests/FSharp.Compiler.Private.Scripting.UnitTests/FSharpScriptTests.fs

```diff
@@ -190,6 +191,24 @@ tInput.Length
         Assert.AreEqual(4L, value.ReflectionValue :?> int64)
 
 
+    [<Test>]
+    member __.``System.Device.Gpio - Ensure we reference the runtime version of the assembly``() =
+        let code = """
+#r "nuget:System.Device.Gpio"
+typeof<System.Device.Gpio.GpioController>.Assembly.Location
+"""
+        use script = new FSharpScript(additionalArgs=[|"/langversion:preview"|])
+        let opt = script.Eval(code)  |> getValue
+        let value = opt.Value
+
+        if RuntimeInformation.IsOSPlatform(OSPlatform.Windows) then
+            Assert.IsTrue( (value.ReflectionValue :?> string).EndsWith(@"runtimes\win\lib\netstandard2.0\System.Device.Gpio.dll") )
+        else if RuntimeInformation.IsOSPlatform(OSPlatform.Linux) then
+            Assert.IsTrue( (value.ReflectionValue :?> string).EndsWith(@"runtimes\linux\lib\netstandard2.0\System.Device.Gpio.dll") )
```

Comment:
This will likely need to be forward slashes.

Reply:
It did.